packages/overlay/src/integrations/sentry/data/sentryDataCache.ts

```diff
@@ -34,7 +34,8 @@ class SentryDataCache {
 
   protected subscribers: Map<string, Subscription> = new Map();
 
-  protected contextLinesProvider: string = new URL(DEFAULT_SIDECAR_URL).origin + CONTEXT_LINES_ENDPOINT;
+  protected sidecarUrl: string = new URL(DEFAULT_SIDECAR_URL).origin;
```

Comment:
Do we need the `new URL().origin` trick with you changing the `DEFAULT_SIDECAR_URL` thingy in the other file?

Reply:
i am removing the logic to extract origin.